src/export/digitalPlanning/model.ts

```diff
@@ -341,10 +342,15 @@ export class DigitalPlanning {
   }
 
   private getProposedBoundary(): Payload["data"]["proposal"]["boundary"] {
+    const annotatedBoundary = this.passport.data?.[
+      "property.boundary.site"
+    ] as unknown as Feature;
+    if (annotatedBoundary && annotatedBoundary.properties)
+      annotatedBoundary["properties"]["planx_user_action"] =
+        this.passport.data?.["drawBoundary.action"];
+
```

Comment:
small schema-related change snuck in here too: we had already mapped the title boundary passport variable to populate `data.property.boundary` & the location plan to populate `data.proposal.boundary` - but this takes it one step further and adds a custom GeoJSON `properties` key onto the `data.proposal.boundary` to explain which action the Planx user took. In cases of amend or accept, `properties` will additionally include the entity source info from Digital Land.

we'll be overdue soon to re-run some example payloads & populate fresh/more realistic example payloads for new features like this.